openfisca_france/parameters/prestations_sociales/aides_logement/allocations_logement/autres/nr_seuil.yaml

```
@@ -8,8 +8,14 @@ values:
     value: 22
   2017-01-01:
     value: 23
+  2021-01-01:
+    value: 24
 metadata:
   unit: currency
   reference:
     2017-01-01:
       href: https://www.legifrance.gouv.fr/affichCodeArticle.do?idArticle=LEGIARTI000035671411&cidTexte=LEGITEXT000006073189&categorieLien=id&dateTexte=
+    2021-01-01:
+      href: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000038929179
+documentation: |-
+  Notes: Seuil calculé au regard du plafond de la sécurité sociale mensuel et arrondi à l'euro supérieur.
```

```suggestion
  Seuil calculé au regard du plafond de la sécurité sociale mensuel et arrondi à l'euro supérieur.
```

---

Une petite précision pourrait être utile (% du PSS ou autre).

---

[Code de la sécurité sociale : Chapitre 3 : Recouvrement des cotisations, versement e](https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000038929179/2022-06-17)
sur cet article je trouve ceci qui semblerait être le % en question :

_Le montant visé à l'article [L. 133-3 ](https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006073189&idArticle=LEGIARTI000006740147&dateTexte=&categorieLien=cid)en deçà duquel les organismes chargés du versement des prestations de sécurité sociale sont autorisés à abandonner la mise en recouvrement des indus de prestations de sécurité sociale versés à leurs assurés ou constatés à l'égard de tiers est fixé à **0,68 % du plafond mensuel de sécurité sociale** en vigueur, arrondi à l'euro supérieur._

```suggestion
  Seuil calculé au regard du plafond de la sécurité sociale mensuel et arrondi à l'euro supérieur (0,68% du plafond).
```

---

Super !
Si on connaît la date de ce changement, on peut le mettre en note à l'année considérée.
Voici un [exemple](openfisca_france/parameters/impot_revenu/calcul_impot_revenu/pv/actifs_numeriques/taux.yaml)